BDKSwiftExampleWallet.xcodeproj/xcshareddata/xcschemes/BDKSwiftExampleWallet.xcscheme

```diff
@@ -57,6 +57,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "fr"
```

Comment:
Let's get rid this default `language = "fr"`, I think maybe its there from editing the scheme to test in French. But we will want it to be System Language probably (I think that's what it was before? At least that's my guess because if you set it to English then this line still gets added but if it's set to System Language then there is no line added).